apps/www/components/ip-whitelisting-bento.tsx

```diff
@@ -1,10 +1,10 @@
-import Image from "next/image";
 import ip from "../images/ip.svg";
+import ImageWithBlur from "./image-with-blur";
```

Comment:
how are you doing default imports if the component is not default exported?

Reply:
Good question